packages/app/src/components/JansSinglePet.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <div
+    class="rounded-md bg-white p-1 px-2 h-full flex flex-col justify-content-start"
+  >
+    <div class="flex-none">
+      <strong>Name: </strong>
+      {{ pet.name }}
+    </div>
+    <div class="flex-none">
+      <strong>Description: </strong>
+      {{ pet.description }}
+    </div>
+    <div class="flex-auto"></div>
+    <div class="flex self-end">
+      <button
+        class="bg-white border border-gray-300 rounded-md px-2 my-1"
+        @click="deleteThisPet"
+      >
+        Remove
+      </button>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue';
+
+import feathers from '../lib/feathers';
```

Comment:
```suggestion
import { defineComponent } from 'vue';
import feathers from '../lib/feathers';
```
Remove line

Reply:
I would separate this as is is external  / internal modules.